src/CppUTest/SimpleString.cpp

```diff
@@ -194,7 +193,7 @@ size_t SimpleString::count(const SimpleString& substr) const
 {
     size_t num = 0;
     char* str = buffer_;
-    while ((str = StrStr(str, substr.buffer_))) {
+    while (*str && (str = StrStr(str, substr.buffer_))) {
```

Comment:
Nevermind above comment, str isn't the substr but the buffer_ It is early morning (5:10am)